package.json

```diff
@@ -61,7 +61,8 @@
     "blockchain.com",
     "blockcypher",
     "reduce",
-    "json-rpc"
+    "json-rpc",
+    "composite/**"
```

Comment:
composite/proof-of-reserves has a lockfile checked in, should remove that if its part of the monorepo